src/test/isolation2/expected/lockmodes.out

```diff
@@ -2214,6 +2214,14 @@ ROLLBACK
 -- when running concurrently with index creation, for Heap tables.
 -- For AO/CO tables, refer to create_index_allows_readonly.source.
 
+-- CBDB_CHERRY_PICK_MERGE_FIXME: uncomment this, when https://github.com/greenplum-db/gpdb-archive/commit/3be17112144 cherry-picked
+-- 1: BEGIN;
+-- 1: INSERT INTO t_lockmods_part_tbl_dml SELECT i, 1, i FROM generate_series(1,10)i;
+-- -- With GDD enabled, QD will only hold lock on root for insert
```

Comment:
this is cherry-picked, need check confilct.

Reply:
So quick! I will post an update then.